app/controllers/admin/base_controller.rb

```diff
@@ -1,7 +1,11 @@
 class Admin::BaseController < ApplicationController
   before_action :require_admin
 
+  private
```

Comment:
I think this can be outside of the private section if we want

Reply:
It won't cause any problems to move it. Do you want it public?